src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/studentlog
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
 spring.datasource.username=root
-spring.datasource.password=root1234
+spring.datasource.password=thisissparta
```

Comment:
?

Reply:
this is my localhost password. I think we should add this application.properties file to .gitignore

Reply:
okay, but why add to this PR then?